src/sentry/options/defaults.py

```diff
@@ -342,3 +342,6 @@
 
 # Try to read release artifacts from zip archives
 register("processing.use-release-archives-sample-rate", default=0.0)
+
+# All Relay options (statically authenticated Relays can be registered here)
+register("relay.static_auth", default={})
```

Comment:
Does this not register the option so that a database lookup could take place? I thought we require FLAG_PRIORITIZE_DISK as flag here

Reply:
hmm... not sure, need to check out exactly what's going on (was hoping that it doesn't access the db).

Reply:
Thanks @mitsuhiko , to make sure it doesn't access the store (which uses ProjectOptionsManager) we should use FLAG_NOSTORE,  FLAG_PRIORITIZE_DISK will also ensure no db access but only if there is something defined in the config file, I'll use both since otherwise I think we'll have a db access for each validation whenever there is no configuration in the file.